reference/sql_workarounds.rst

```diff
@@ -0,0 +1,141 @@
+.. _workarounds:
+
+SQL Workarounds
+===============
+
+Citus supports most, but not all, SQL statements directly. Its SQL support continues to improve. Also many of the unsupported features have workarounds; below are a number of the most useful.
+
+Subqueries
+----------
+
+In analytics it is common to model information using a "star schema." This is a collection of dimensional tables (e.g. observations, statistics) referencing a table of facts (e.g. people, pages or events). Furthermore, a common form of query examines dimensions for a subset of the facts. Examples of these kind of queries:
```

Comment:
I see now how fact-dimension is the wrong scenario because it can be accomplished best by broadcasting the small table. @saicitus if you can email me examples of the queries for the scenarios listed in this document then I'll have a better idea of the actual schemas and can write more accurately.